test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestJps.java

```diff
@@ -100,8 +100,9 @@ public void testApplicationArguments() throws IOException {
 		TargetManager tgtMgr = new TargetManager(TARGET_VM_CLASS, null, targetArgs);
 		assertTrue(CHILD_PROCESS_DID_NOT_LAUNCH, tgtMgr.syncWithTarget());
 		List<String> jpsOutput = runCommandAndLogOutput(Collections.singletonList("-m")); //$NON-NLS-1$
-		Optional<String> searchResult = StringUtilities.searchSubstring(tgtMgr.targetId, jpsOutput);
-		assertTrue(CHILD_IS_MISSING, StringUtilities.searchSubstring(tgtMgr.targetId, jpsOutput).isPresent());
+		String needle = tgtMgr.targetId + " TargetVM";
+		Optional<String> searchResult = StringUtilities.searchPrefixSubstring(needle, jpsOutput);
+		assertTrue(CHILD_IS_MISSING, StringUtilities.searchPrefixSubstring(needle, jpsOutput).isPresent());
```

Comment:
This should be `searchResult.isPresent()`, same for the next change.

Reply:
Updated.